src/benchmarks/micro/corefx/System.Diagnostics/Perf_Process.cs

```diff
@@ -22,13 +22,6 @@ public class Perf_Process
         [Benchmark]
         public void GetProcessById() => Process.GetProcessById(_currentProcessId).Dispose();
 
-        [Benchmark]
-        public void EnterLeaveDebugMode()
```

Comment:
@jorive @billwert I am removing this benchmark because:

1. it does nothing on Unix OSes (not implemented in CoreCLR, instead of throwing does nothing)
2. On Windows it requires some certain privileges to work, it currently fails even on my box (it used to work fine)

I am the person who created this benchmark ;)